docusaurus/src/components/RequestERD/EmailModal.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from "react";
+import styles from "./EmailModal.module.css";
+import { faXmark } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import cn from "classnames";
+import { useDebouncedCallback } from "use-debounce";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+
+const CloseButton = ({ onClose }) => {
+  return (
+    <button onClick={onClose} className={styles.modalContentClose}>
+      <FontAwesomeIcon icon={faXmark} />
+    </button>
+  );
+};
+
+const ModalBody = ({ status, onSubmit }) => {
+  const [email, setEmail] = useState("");
+  const [emailError, setEmailError] = useState("");
+
+  const debouncedValidation = useDebouncedCallback((value) => {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    if (emailRegex.test(value)) {
+      setEmailError("");
+    } else {
+      setEmailError("Please enter a valid email address");
+    }
+  }, 800);
+
+  const handleOnChange = (e) => {
+    const emailValue = e.target.value;
+    setEmail(emailValue);
+    debouncedValidation(emailValue);
+  };
+
+  if (status === "success") {
+    return (
+      <div>
+        <p> Request submitted successfully!</p>
+        <p> We will notify you when the ERD is ready</p>
+      </div>
+    );
+  }
+  if (status === "error") {
+    return (
+      <div>
+        <p> Error submitting request. Please try again later.</p>
+      </div>
+    );
+  }
+
+  return (
+    <form onSubmit={() => onSubmit(email)} className={styles.modalContentForm}>
+      <div className={styles.modalContentInputContainer}>
+        <input
+          className={cn(styles.modalContentInput, {
+            [styles.modalContentInputError]: emailError,
+          })}
```

Comment:
It would look more consistent with the rest of our UI if we changed the border of the input to red, and kept the text color the same, when this input is in an error state.